cmd/tuf/remove_key.go

```diff
@@ -0,0 +1,39 @@
+package main
+
+import (
+	"os"
+	"path/filepath"
+
+	"github.com/flynn/go-docopt"
+	"github.com/theupdateframework/go-tuf"
+)
+
+func init() {
+	register("remove-key", cmdRemoveKey, `
+usage: tuf remove-key [--expires=<days>] <role> <id>
+
+Remove a signing key
+
+Before the key is removed the key will be first revoked
+The key will then be removed from the root metadata file and if the key is present in 
+"keys" directory it will also be removed 
+
+Options:
+  --expires=<days>   Set the root metadata file to expire <days> days from now.
```

Comment:
The `expires` value is not used anywhere later.

Reply:
What I meant is since you also revoke the key as part of the process it should be expected to take into account the expiration value too, if present.

Reply:
Just to make sure I understand exactly is that that code should be able to extract the --expires option value and then update the expiration value if --expires is provided right?

Reply:
It should work fine, yes 👍